velox/docs/functions/presto/conversion.rst

```diff
@@ -0,0 +1,269 @@
+====================
+Conversion Functions
+====================
+
+During expression evaluations, Velox doesn't implicitly convert expression
+arguments to the correct types. If such a conversion is necessary, two provided
+conversion functions can be used explicitly cast values to a particular type.
+
+Conversion Functions
+--------------------
+
+.. function:: cast(value AS type) -> type
+
+    Explicitly cast a value as a type. This can be used to cast a varchar to a
+    numeric value type and vice versa.
+
+.. function:: try_cast(value AS type) -> type
+
+    Like :func:`cast`, but returns null if the cast fails. ``try_cast(x AS type)``
+    is different from ``try(cast(x AS type))`` in that when there is an error
+    during the evaluation of ``x``, ``try_cast`` throws, while ``try(cast())``
+    suppresses this error and produces a NULL result at the current row row.
+
+Supported Conversions
+---------------------
+
+The supported conversions in Velox are listed below. Conversions of ARRAY, MAP,
+and ROW types are supported if the conversion of their element types are
+supported. In addition to this list, supported conversions to/from JSON are
+listed in :doc:`json`
+
+.. list-table::
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - Return types
+     - Argument types
+     - Truncate support
+   * - tinyint
+     - tinyint
+     - N
+   * - tinyint
```

Comment:
This table is long and a bit hard to read. I wonder if some different layout will work better. Maybe nxn matrix with from types in rows and to types in columns? Maybe a few matrices?